.github/workflows/osx.yml

```diff
@@ -34,13 +33,28 @@ jobs:
       - name: configure
         run: |
           cd build
-          cmake -DBUILD_TESTING=ON -DCMAKE_PREFIX_PATH=/usr/local/ -DUSE_OPENMP=OFF -DUSE_HDF5=ON -DBUILD_PYTHON=OFF -DBUILD_PYTHON3=ON -DBoost_NO_BOOST_CMAKE=True -DDATA_DIR=. ..
+          cmake \
+            -DBUILD_TESTING=ON \
+            -DCMAKE_PREFIX_PATH=/usr/local/ \
+            -DUSE_OPENMP=OFF \
+            -DUSE_HDF5=ON \
+            -DBUILD_PYTHON=OFF \
+            -DBUILD_PYTHON3=ON \
+            -DBoost_NO_BOOST_CMAKE=True \
+            -DCMAKE_Fortran_COMPILER=gfortran-10 \
+            -DDATA_DIR=. ..
 
       - name: build
-        run: make -j3
+        run: |
+          cd build
+          make -j
```

Comment:
i think this still forks unlimited processes, which might slow down the compilation. the github actions documentation states that the osx vm's have 3 cores available.

Reply:
ah ok, feel free to change it back. For me, it was 40 minutes with `-j3` and 37 with `-j`